enterprise/config/make_config.py

```
@@ -0,0 +1,177 @@
+    print("\n[yellow]Next steps:[/yellow]")
+    print("1. Review the generated docker-compose.yml file, and adjust it if needed")
+    if config.get("use_custom_cert"):
+        print("2. Ensure your certificate files are in place:")
+        print(f"   - Certificate: {config['cert_config']['cert_path']}")
+        print(f"   - Private key: {config['cert_config']['key_path']}")
+    if config["db"] == "postgresql":
+        print("3. Make sure PostgreSQL passwords are properly set")
+    if config["need_mailer"]:
+        print("4. Verify email configuration settings")
+    print(
+        f"5. Run './docker-compose.sh' and follow the instructions to create the first admin user"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)